experimental/dds/tree2/src/feature-libraries/schemaBuilder.ts

```diff
@@ -51,25 +67,29 @@ export class SchemaBuilder<
 	 * Reduced type safety is a side effect of a workaround for a TypeScript limitation.
 	 *
 	 * See note on RecursiveTreeSchema for details.
```

Comment:
Should we link to `RecursiveTreeSchema` here, since this method shows up in the public API?

Reply:
(same with the other methods)

Reply:
done